Dockerfile

```diff
@@ -57,6 +57,11 @@ RUN cd ${PROJECT_PATH}
 # Export benchmarks results as a volume
 VOLUME ${BENCHMARKS_OUTPUT_PATH}
 
+# Add script to test TCP connections of external services that are specified
+# as dependencies for the runner container
+COPY wait-for.sh /usr/local/bin
+RUN ln -s usr/local/bin/wait-for.sh / # backwards compat
```

Comment:
What are we waiting for aka which service? I mean we need a way to specify services or ports to wait for right or am I missing something?

Reply:
excuse this of course just links that makes everything clear, I'm too tired to be reviewing stuff I think now :)